paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetAddPaymentMethodFragmentTest.kt

```diff
@@ -97,7 +97,7 @@ internal class PaymentSheetAddPaymentMethodFragmentTest : PaymentSheetViewModelT
         }
         createFragment(stripeIntent = paymentIntent) { _, viewBinding, _ ->
             val item = viewBinding.paymentMethodsRecycler.layoutManager!!.findViewByPosition(0)
-            assertThat(item!!.measuredWidth).isEqualTo(104)
+            assertThat(item!!.measuredWidth).isEqualTo(116)
```

Comment:
Why did these values change?

Reply:
Previously we were measuring just the card part of the view. Now we have to measure the whole compose view for this test, which now includes (116-104) 12dp of padding